app/models/spree/shipment_decorator.rb

```diff
@@ -1,7 +1,7 @@
 Spree::Shipment.class_eval do
   def self.exportable
-    query = joins(:order).where(spree_orders: { state: 'complete' })
-    query = query.where.not(spree_shipments: { state: 'pending' }) if Spree::Config.require_payment_to_ship
+    query = joins(:order).merge(Spree::Order.complete)
+    query = query.where.not(spree_shipments: { state: 'pending' }) if !Spree::Config.shipstation_capture_at_notification
```

Comment:
I think we should be doing:

```ruby
query = query.where(spree_shipments: { state: 'ready' }) unless Spree::Config.shipstation_capture_at_notification 
```

Otherwise you might select `canceled` shipments, even better we can do:

```rb
query = query.ready unless Spree::Config.shipstation_capture_at_notification 
```